pages/pricing/_pricing-patcher.html

```diff
@@ -0,0 +1,52 @@
+<ul>
+  <li class="grid-container text-bold text-white">
+    <span
+      class="light"
+      title=""
+      data-toggle="tooltip"
+      >Patcher</span
```

Comment:
One thing I missed in the design: Patcher should say "Beta" next to it on the pricing page, just like it does everywhere else on the site.